packages/flutter/lib/src/material/text_field.dart

```diff
@@ -339,7 +346,10 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
       return effectiveDecoration;
 
     final int currentLength = _effectiveController.value.text.runes.length;
-    final String counterText = '$currentLength/${widget.maxLength}';
+    final String counterText = '$currentLength' +
```

Comment:
This will also either require accessibility localizations

Reply:
Ah -- I have no idea what "accessibility localizations" actually me since the old code obviously didn't have them either. Do you have a pointer as to what that is?

Reply:
Currently when talkback is enabled we read out the the number of remaining characters. It doesn't make sense to do that if there is no limit, so you'll want to make sure semanticCounterText is empty